terraform/gcp/k8s/admin-deployment.yml

```diff
@@ -50,23 +52,24 @@ spec:
               # The proxy's CPU use scales linearly with the amount of IO between
               # the database and the application. Adjust this value based on your
               # application's requirements.
-              cpu:    "1"
+              cpu: "1"
 
         - name: admin
-          image: us-central1-docker.pkg.dev/broad-ddp-dev/juniper/juniper-admin:1.3.56
+          image: us-central1-docker.pkg.dev/broad-ddp-dev/juniper/juniper-admin:1.3.75
           env:
+            - name: SPRING_PROFILES_ACTIVE
+              value: gcp
+            - name: DB_NAME_SECRET_ID
+              value: "d2p-db-name"
+            - name: DB_PASSWORD_SECRET_ID
+              value: "d2p-db-password"
+            - name: DB_USER_SECRET_ID
+              value: "d2p-db-user"
```

Comment:
it feels like it might be one extra layer of indirection to store the names of the secrets as environment variables.  But I guess it could come in handy for repointing to a different DB

Reply:
i figured if somehow we need to have different names for dev/prod it would be nice, plus I wanted to avoid hardcoding any implementation-specific details like the names of things if possible, but happy to go either way with this